README.md

```diff
@@ -13,13 +13,14 @@ Python 2.7.5
 ## Running the script
 
 ```
-usage: verifyFiles.py [-h] [-c] [-d FILEDIFF] [-m METRICSFILE] file_in
+usage: verifyFiles.py [-h] [-c] [-d FILEDIFF] [-m METRICSFILE]  [-e [EXCLUDEFOLDERS [EXCLUDEFOLDERS ...]]] file_in
 
 file_in,                                   Mandatory parameter.Directory path to the file containing the conditions.
 -h, --help                                 Show this help message and exit
 -c, --checksumFile                         Reads checksum from files. Doesn't compute checksums locally if this parameter is set.
 -d FILEDIFF, --fileDiff FILEDIFF           Writes the difference matrix into a file.
 -m METRICSFILE, --metricsFile METRICSFILE  CSV file containing metrics definition. Every line contains 4 elements: metric_name,file_extension,command_to_run,output_file_name
+-e EXCLUDEFOLDERS, --excludeFolders        Name of the folders which should be excluded from creating checksums. 
```

Comment:
I tested it with subject folder and folders inside the subject. For eg: "unprocessed" . It ignores the entire contents of the given directory under every condition. I will test it with a condition folder and see how it works.